src/client/connect/http.rs

```diff
@@ -27,6 +27,7 @@ use super::{Connected, Connection};
 ///
 /// Sets the [`HttpInfo`](HttpInfo) value on responses, which includes
 /// transport information such as the remote socket address used.
+#[cfg_attr(docsrs, doc(cfg(feature = "tcp")))]
```

Comment:
I believe this attribute is inherited from it being on the `client::connect::http` module, which you also added.

Reply:
I agree that it should be! But for some reason this is what happens if you don't manually add this attribute:

<img width="992" alt="image" src="https://user-images.githubusercontent.com/882976/103243068-8a21fc00-4926-11eb-890d-be727a093272.png">

The "requires `tcp` feature" annotation gets added to all the methods, but not the type itself. I didn't dig into why, since adding the feature manually gave the results I expected:

<img width="1011" alt="image" src="https://user-images.githubusercontent.com/882976/103243109-ba699a80-4926-11eb-8682-237115aca380.png">

Reply:
Weird, but thanks for checking!